### ISSUE TITLE
Load SSL certificate from WebUI

### ISSUE THREAD

**Issue by kenkendk** [duplicati/duplicati#1843]
With the SSL support for the webserver in #1829 related to issue #1198, the SSL certificate should be loadable from the WebUI.

**Comment by mnaiman** [comment]
@HRomie yeah, this is what I think to do, if user select in settings that wants SSL, generate self signed certificate with possibility to change.
Definitely at least "valid to" should be displayed in web ui.
